Source/AppLocker/AppLocker.swift

```diff
@@ -18,12 +18,15 @@ public enum ALConstants {
     
     enum Button: Int {
         case delete = 1000
-        case cancel = 1001
+        case forgot = 1001
+        case cancel = 1002
     }
 }
 
-public typealias OnSuccessfulDismissCallback = (_ mode: ALMode?, _ pin: String?) -> () // Cancel dismiss will send mode as nil
-public typealias OnFailedAttemptCallback = (_ mode: ALMode) -> ()
+public typealias OnSuccessfulDismissCallback = (_ mode: ALMode?, _ pin: String?) -> Void // Cancel dismiss will send mode as nil
```

Comment:
Having multiple unnamed parameters reduces readability at when calling the function

Reply:
sure, will fix in next PR